src/node.cc

```diff
@@ -5266,16 +5266,11 @@ v8::Local<v8::Value> Evaluate(const std::string& java_script_code) {
   MaybeLocal<v8::Script> script = v8::Script::Compile(env->context(), v8::String::NewFromUtf8(isolate, java_script_code.c_str())/*, origin*/);
   if (script.IsEmpty()) {
     ReportException(env, try_catch);
-    exit(3); //TODO jh: don't exit process when function breaks. Handle error differently.
+    return MaybeLocal<v8::Value>();
   }
 
-  Local<Value> result = script.ToLocalChecked()->Run();
-  if (result.IsEmpty()) {
-    ReportException(env, try_catch);
```

Comment:
What did the `try_catch` do? If it's not needed any longer, remove it from line 5259.

Reply:
To be honest with you, i dont know exactly what the `TryCatch` does. However, it is also used in line 5267 (`if (script.IsEmpty())`). Therefore, i would not remove it entirely (at least not until we figured out more about it).